README.md

```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
 
 https://github.com/user-attachments/assets/c1c3b7da-9f4d-4ddc-b531-02a951505b03
 
+Note: This project was developed as part of a hackathon. While we strived to deliver a functional and engaging application, the code quality may not meet production standards due to time constraints.
+
+
+# 📚 Overview
+The Landsat Web Tracker bridges the gap between ground-based spectral measurements and Landsat Surface Reflectance (SR) data. By providing users with an intuitive web-based interface, the application makes satellite data more accessible and actionable, fostering environmental awareness and empowering individuals to contribute toward sustainability.
+Our solution allows users to:
+
+- Select a target location and time range.
+- Receive notifications when Landsat satellites pass over their chosen location.
+- Access and visualize Landsat data, including key environmental metrics like temperature, cloud coverage, and Earth-Sun distance.
+- Generate reports with advanced metadata, band views, and reflectance analyses.
+- Download data in PDF and CSV formats.
```

Comment:
That's not entirely true

Reply:
Umm, so let's not spread this lie even further :3

Reply:
ok so tell me which stuff is true and which is not